docs/InstrumentationVerification/InstrumentationVerification.md

```diff
@@ -0,0 +1,134 @@
+# Instrumentation Verification Library
+
+
+# Description
+
+The Instrumentation Verification library allows us to generate an assembly on disk which contains all the metadata and bytecode changes that were performed at runtime through the CLR Profiling API, and then run that assembly through various tools that verify the correctness of the instrumented assembly.
+
+
+The library is meant to be used in several distinct two-cases:
+
+
+
+1. As part of CI, the tool can be used:
+    1. By the Tracer team, to verify that we produce valid IL for every integration we add
+    2. By the Debugger team, to verify that adding probes produces valid IL on a wide range of methods
+2. As a diagnostic tool, the InstrumentationVerification tool can be used to quickly triage support cases where the customer is complaining that our CLR Profiler causes a `BadImageFormationException`, `InvalidProgramException`, `TypeLoadException`, etc etc. 
```

Comment:
```suggestion
2. As a diagnostic tool, the InstrumentationVerification tool can be used to quickly triage support cases where the customer is complaining that our CLR Profiler causes a `BadImageFormatException`, `InvalidProgramException`, `TypeLoadException`, etc etc. 
```

Reply:
Fixed in 4c7c44dcf9bc4f5affbde6c20b8fd17d1735d225